core/arch/arm/plat-imx/imx-regs.h

```diff
@@ -36,12 +36,13 @@
 #include <registers/imx7.h>
 #elif defined(CFG_MX7ULP)
 #include <registers/imx7ulp.h>
-#elif defined(CFG_MX8MQ) || defined(CFG_MX8MM) || defined(CFG_MX8MN)
+#elif defined(CFG_MX8MQ) || defined(CFG_MX8MM) || defined(CFG_MX8MN) || \
+	defined(CFG_MX8MP)
 #include <registers/imx8m.h>
 #elif defined(CFG_MX8QX) || defined(CFG_MX8QM)
 #include <registers/imx8q.h>
 #else
-#error "CFG_MX6/7/7ULP or CFG_MX8MQ/8MM/8QX/8QM is not defined"
+#error "CFG_MX6/7/7ULP or CFG_MX8MQ/8MM/8MN/8MP/8QX/8QM is not defined"
```

Comment:
`8MN` duplicated.

Reply:
I see only one `8MN` in this line.
This patch also adds `8MN` that was supposed to be added withc the `8MN` support.

Reply:
sorry, missed it. sure fine,
`MM`/`MN` confused me. thanks @jforissier for not waiting my ack :+1: